openwisp_monitoring/device/tests/__init__.py

```diff
@@ -35,6 +35,23 @@ def _post_data(self, id, key, data):
     def _data(self):
         return {
             'type': 'DeviceMonitoring',
+            'general': {
+                'local_time': 1589026500,
+                'uptime': 8003,
+                'hostname': '08-00-27-04-8E-1C',
+            },
+            'resources': {
+                'memory': {
+                    'total': 249774080,
+                    'shared': 90112,
+                    'free': 224497664,
+                    'cached': 6774784,
+                    'available': 203497472,
```

Comment:
available and shared are not expected in the current schema. Same for general->hostname, can you update them in one different PR?

Reply:
I copied the actual device data, funny that schema doesn't have `shared` because that is being displayed in `Device Status` :sweat_smile:

Reply:
Yeah the current schema is somewhat flexible and remember e.g. that we were not forcing format checking but as it gets more stable and we are confident with it, IMO we should get stricter and reject metrics that don't follow the schema